### PR TITLE
Better gh actions

### PR DESCRIPTION
**Pull request by barbariand**
We can add this once #24 is merged

### CODE REVIEW

File: .github/workflows/reti-js.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Rust
+
+on:
+  push:
+    paths:
+      - Reti-js/**
+  pull_request:
+    paths:
+      - Reti-js/**
```
Comment (Alvinn8) [inline .github/workflows/reti-js.yml]:
```suggestion
      - reti-js/**
  pull_request:
    paths:
      - reti-js/**
```

**Review by Alvinn8 — COMMENTED** [review]
Maybe we should run tests in parser.yml, repl.yml, etc.? Now they will only fail if we have compile errors right? But it might also be good to always run tests (like test.yml does) since changes in one module can still affect others

**Comment by barbariand** [comment]
Yeah I think the tests I have rn are the like run all test

**Comment by barbariand** [comment]
If anything then we should add like parser-test actions and sutch but keep the test as running all tests in workspace

**Comment by barbariand** [comment]
As #24 is now merged we can merge this one when it passes

**Comment by Alvinn8** [comment]
@barbariand can you rebase this PR? Something is weird with the commits, so I think it just needs to be rebased maybe. Then we can merge this since the github actions will need to change/be modified when we add the svelte projects so it would be nice to already have these changes in dev.

**Comment by barbariand** [comment]
Im not home for like 5 hours so i cant rn if it is not done when i get back i will do it

**Comment by Alvinn8** [comment]
Ok, no problem!

**Comment by Alvinn8** [comment]
I rebased and merged this PR now

**Comment by barbariand** [comment]
Nice i had more things i needed to do

**Comment by barbariand** [comment]
Sadly

**Pull request merged**